### ISSUE TITLE
eol.php layout broken on mobile due to extra-wide table

### ISSUE THREAD

**Issue by SteveDesmond-ca** [php/web-php#498]
Steps to reproduce:
- browse to https://www.php.net/eol.php with a viewport <=360px
- scroll to the right to see the notes for past releases

Expected result:
- notes are legible

Observed result:
- wide table breaks intended page layout
![Screen Shot 2022-02-24 at 12 40 17](https://user-images.githubusercontent.com/195896/155577967-39ab5dc8-513d-47a9-847e-adac810a1fcb.png)

**Comment by cmb69** [comment]
Oh, that dark bar at the right hand side is unlikely to be intended here. I also see it with wider viewports (about 950px).

**Comment by SteveDesmond-ca** [comment]
Yeah, I see that too, between 768-979px; but I think that's a separate problem, where the sidebar shouldn't be shown if there's no `aside`. The `width: 70% !important` in the media query is overriding the `:only-child` selector which would otherwise have the main conent be full-width. Want me to create another issue for that one?

**Comment by derickr** [comment]
I can't reproduce this — there have been changes, so perhaps this is fixed?

**Comment by SteveDesmond-ca** [comment]
Still there to some extent: shrink the browser's viewport to 320px wide and scroll right.
![Screen Shot 2023-11-29 at 12 17 44](https://github.com/php/web-php/assets/195896/9f7c83cb-9c7a-46a2-9998-294c2d36f462)

I believe the best fix at this point is to set the `thead` and "odd" rows' (`2n+1`) `background-color` to `var(--content-background-color)` instead of the default `transparent`.
![Screen Shot 2023-11-29 at 12 30 20](https://github.com/php/web-php/assets/195896/07fd95de-219e-4865-91b2-8e58e8b688db)

```css
table.standard tr:nth-child(2n+1) td, table.standard thead th {
  background-color: var(--content-background-color);
}
```
That said, the fix from #499 decreased the breaking width to 352px, and the vast majority of phones these days are wider than that, so I'd be willing to have this closed as "good enough" since very few people will still experience this.
